ui/desktop/tests/acceptance/cluster-url-test.js

```diff
@@ -115,6 +111,8 @@ module('Acceptance | clusterUrl', function (hooks) {
 
   test('visiting index without a clusterUrl specified redirects to clusterUrl route', async function (assert) {
     assert.expect(2);
+    this.owner.register('service:browser/window', WindowMockIPC);
```

Comment:
What is the reason why we delete this from the `beforeEach` statement and move it to each test? I assume it is because not all test in the test file use it?

If the reason is not all test use it, I think we could extract this code to a function before the `describe` block, then, each test that needs the mock of the `service:browser/window` just call that function.

Let me know your thoughts :wink:

Reply:
@calcaide absolutely